packages/client/src/app.js

```diff
@@ -152,6 +156,7 @@ app.use((state, emitter) => {
 
 app.use(messages.store)
 app.use(sidebar.store)
+app.use(userList.store)
 app.use(accountSettings.store)
```

Comment:
aaaa I should really get started on fixing data flow in the app because this is bad